base/applications/network/ipconfig/lang/ro-RO.rc

```diff
@@ -3,45 +3,46 @@
  * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
  * PURPOSE:     Romanian resource file
  * TRANSLATOR:  Copyright 2011-2018 Ștefan Fulea <stefan.fulea@mail.com>
+ * TRANSLATOR:  Copyright 2023 Andrei Miloiu <miloiuandrei@gmail.com>
  */
 
 LANGUAGE LANG_ROMANIAN, SUBLANG_NEUTRAL
 
 STRINGTABLE
 BEGIN
-    IDS_HEADER "\nReactOS IP Configuration\n\n"
-    IDS_HOSTNAME            "\tHost Name . . . . . . . . . . . . : %hs\n"
-    IDS_PRIMARYDNSSUFFIX    "\tPrimary DNS Suffix. . . . . . . . : %hs\n"
-    IDS_NODETYPE            "\tNode Type . . . . . . . . . . . . : "
-    IDS_IPROUTINGNO         "\tIP Routing Enabled. . . . . . . . : No\n"
-    IDS_IPROUTINGYES        "\tIP Routing Enabled. . . . . . . . : Yes\n"
-    IDS_WINSPROXYNO         "\tWINS Proxy enabled. . . . . . . . : No\n"
-    IDS_WINSPROXYYES        "\tWINS Proxy enabled. . . . . . . . : Yes\n"
-    IDS_DNSSUFFIXLIST       "\tDNS Suffix Search List. . . . . . : %hs\n"
+    IDS_HEADER "\nConfigurația IP ReactOS IP\n\n"
+    IDS_HOSTNAME            "\tNumele gazdei . . . . . . . . . . . . : %hs\n"
+    IDS_PRIMARYDNSSUFFIX    "\tSufixul primar DNS. . . . . . . . : %hs\n"
+    IDS_NODETYPE            "\tTipul nodului . . . . . . . . . . . . : "
+    IDS_IPROUTINGNO         "\tRutarea IP activată. . . . . . . . : Nu\n"
+    IDS_IPROUTINGYES        "\tRutarea IP activată. . . . . . . . : Da\n"
+    IDS_WINSPROXYNO         "\tProxy-ul WINS activat. . . . . . . . : Nu\n"
+    IDS_WINSPROXYYES        "\tProxy-ul WINS activat. . . . . . . . : Da\n"
+    IDS_DNSSUFFIXLIST       "\tLista de căutare a sufixului DNS. . . . . . : %hs\n"
```

Comment:
If you look at the en-US localization, you will see that the text is aligned by the semicolon character ":", which is not the case within your changes. This has to be fixed for all following strings.